app/services/device_tracking/manage_device.rb

```diff
@@ -0,0 +1,36 @@
+module DeviceTracking
+  class ManageDevice
+    # primary method: updates the usage time if it is not a new device, and calls create
```

Comment:
Is this comment necessary? Do you think a person reading this wouldn't be able to ascertain this from reading the code? If so, maybe we should change the code to be more clear?

Reply:
The code is clear I think. The purpose of the comment was to ensure anyone who sees it understands that the lack of alerting if no devices exist was on purpose (?) I removed it however.